app/models/annotation/nml/NmlWriter.scala

```diff
@@ -329,7 +329,7 @@ class NmlWriter @Inject()(implicit ec: ExecutionContext) extends FoxImplicits {
     }
     Xml.withinElementSync("meta") {
       writer.writeAttribute("name", "timestamp")
-      writer.writeAttribute("content", DateTime.now().getMillis.toString)
+      writer.writeAttribute("content", Instant.now.epochMillis.toString)
```

Comment:
I've seen this twice now, maybe add another method on Instant for this? (Not sure if it comes another time, if not, also fine as-is.)